packages/core/src/layout/ItemCard/ItemCard.tsx

```diff
@@ -30,6 +31,9 @@ const useStyles = makeStyles(theme => ({
     overflow: 'hidden',
     textOverflow: 'ellipsis',
   },
+  withTags: {
+    height: 'calc(175px - 32px - 8px)',
```

Comment:
Do these numbers have meanings? Can they be variables? The calc seems unnecessary as it is - if it can just be `135px` and a comment can explain why.

Reply:
Yes, the are related to the paddings in `content` and `header`, but not sure how I could pull this into variables as `theme.spacing` is used there and the spacing is not a pixel value.